### PR TITLE
Task/14274 Adapt manage your tests tile

### PR DESCRIPTION
**Pull request by flxschmidt**
## Description
Adapt `HomeTestRegistrationTableViewCell` compliant to Sigma Design Requirements

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14274

## Screenshots
Screen Shot 2022-11-03 at 18 06 54

### CODE REVIEW

**Review by CV113 — APPROVED** [review]
UA approved

**Review by puneetmahali1 — APPROVED** [review]
LGTM!

File: src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertifiedPerson/Cells/HealthCertificate/HealthCertificateCellViewModel.swift [hunk 1]
```diff
@@ -189,6 +189,8 @@ final class HealthCertificateCellViewModel {
 			return UIImage(named: "Icon_CurrentlyUsedCertificate_green")
 		case .blueRedTilted, .blueOnly, .solidGrey, .whiteWithGreyBorder, .whiteToLightBlue:
 			return UIImage(named: "Icon_CurrentlyUsedCertificate_grey")
+		default:
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertifiedPerson/Cells/HealthCertificate/HealthCertificateCellViewModel.swift:192]: Why do we need this change?

File: src/xcode/ENA/ENA/Source/Scenes/Home/Cells/TestRegistration/HomeTestRegistrationTableViewCell.swift [hunk 2]
```diff
@@ -50,7 +58,10 @@ final class HomeTestRegistrationTableViewCell: UITableViewCell {
 
 	// MARK: - Private
 
+
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Scenes/Home/Cells/TestRegistration/HomeTestRegistrationTableViewCell.swift]: remove extra space

```suggestion
```

File: src/xcode/ENA/ENA/Source/Views/GradientView.swift [hunk 3]
```diff
@@ -251,11 +250,6 @@ class GradientView: UIView {
 			gradientLayer.borderWidth = 2
 			
 		case .whiteToLightBlue:
-			var isDarkMode: Bool = false
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Views/GradientView.swift:254]: 👍

File: src/xcode/ENA/ENA/Source/Views/GradientView.swift [hunk 4]
```diff
@@ -61,6 +61,7 @@ class GradientView: UIView {
 		case green
 		case whiteWithGreyBorder
 		case whiteToLightBlue
+		case bottomLightBlueToTopWhite
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Views/GradientView.swift]: Just wondering if we remove Bottom and top here, as we have not used the orientations in enums above

File: src/xcode/ENA/ENA/Source/Scenes/Home/Cells/TestRegistration/HomeTestRegistrationTableViewCell.swift [hunk 5]
```diff
@@ -20,6 +20,12 @@ final class HomeTestRegistrationTableViewCell: UITableViewCell {
 
 		setup()
 	}
+	
+	override func layoutSubviews() {
+		super.layoutSubviews()
+		
+		gradientView.roundCorners(corners: [.topLeft, .topRight], radius: 16)
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Scenes/Home/Cells/TestRegistration/HomeTestRegistrationTableViewCell.swift:27]: Do we then need to reset the corners in init of gradientView for reuse?

File: src/xcode/ENA/ENA/Source/Views/GradientView.swift [hunk 6]
```diff
@@ -96,6 +97,14 @@ class GradientView: UIView {
 	// MARK: - Private
 
 	private let imageView: UIImageView = UIImageView()
+	
+	private var isDarkMode: Bool {
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Views/GradientView.swift:101]: 👍

File: src/xcode/ENA/ENA/Source/Views/GradientView.swift (hunk 4 above)
Comment (flxschmidt) [reply src/xcode/ENA/ENA/Source/Views/GradientView.swift]: Yes I also don't like that naming. Especially that's just the light-gradient colors. But there is noch given name from the design specs. Feel free to make a better suggestion :)

File: src/xcode/ENA/ENA/Source/Scenes/Home/Cells/TestRegistration/HomeTestRegistrationTableViewCell.swift (hunk 5 above)
Comment (flxschmidt) [reply src/xcode/ENA/ENA/Source/Scenes/Home/Cells/TestRegistration/HomeTestRegistrationTableViewCell.swift:27]: The cell is used once, without reuse.

File: src/xcode/ENA/ENA/Source/Views/GradientView.swift (hunk 4 above)
Comment (naveeddotio) [reply src/xcode/ENA/ENA/Source/Views/GradientView.swift]: Just lightBlueToWhite?

File: src/xcode/ENA/ENA/Source/Scenes/Home/Cells/TestRegistration/HomeTestRegistrationTableViewCell.swift (hunk 5 above)
Comment (naveeddotio) [reply src/xcode/ENA/ENA/Source/Scenes/Home/Cells/TestRegistration/HomeTestRegistrationTableViewCell.swift:27]: @flxschmidt But this gradient view is used at various places in the app, by reseting in the init of GradientView we make sure that nothing it is expected.

Comment (flxschmidt) [reply src/xcode/ENA/ENA/Source/Scenes/Home/Cells/TestRegistration/HomeTestRegistrationTableViewCell.swift:27]: `roundCorners(corners:radius:)` is an extension of `UIView` ;)
It just masks the UIView layer with round corners... and only for the unique instance of that View. Here: a gradient view.

File: src/xcode/ENA/ENA/Source/Views/GradientView.swift (hunk 4 above)
Comment (flxschmidt) [reply src/xcode/ENA/ENA/Source/Views/GradientView.swift]: sounds good

**Review by naveeddotio — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/Home/Cells/TestRegistration/HomeTestRegistrationCellModel.swift [hunk 7]
```diff
@@ -9,10 +9,12 @@ class HomeTestRegistrationCellModel {
...
 
 	var title = AppStrings.Home.TestRegistration.title
+	var subtitle = AppStrings.Home.TestRegistration.subtitle
 	var description = AppStrings.Home.TestRegistration.description
 	var buttonTitle = AppStrings.Home.TestRegistration.button
-	var image = UIImage(named: "Illu_Hand_with_phone-initial")
+	var image = UIImage(named: "Illu_WarningAfterSelfTest")
 	var tintColor: UIColor = .enaColor(for: .textPrimary1)
-	var accessibilityIdentifier = AccessibilityIdentifiers.Home.submitCardButton
+	var gradientViewType: GradientView.GradientType = .bottomLightBlueToTopWhite
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Scenes/Home/Cells/TestRegistration/HomeTestRegistrationCellModel.swift]:
```suggestion
	var gradientViewType: GradientView.GradientType = . lightBlueToWhite 
```

File: src/xcode/ENA/ENA/Source/Scenes/Home/Cells/TestRegistration/HomeTestRegistrationCellModel.swift [hunk 8]
```diff
@@ -9,10 +9,12 @@ class HomeTestRegistrationCellModel {
...
 
 	var title = AppStrings.Home.TestRegistration.title
+	var subtitle = AppStrings.Home.TestRegistration.subtitle
 	var description = AppStrings.Home.TestRegistration.description
 	var buttonTitle = AppStrings.Home.TestRegistration.button
-	var image = UIImage(named: "Illu_Hand_with_phone-initial")
+	var image = UIImage(named: "Illu_WarningAfterSelfTest")
 	var tintColor: UIColor = .enaColor(for: .textPrimary1)
-	var accessibilityIdentifier = AccessibilityIdentifiers.Home.submitCardButton
+	var gradientViewType: GradientView.GradientType = . lightBlueToWhite
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Scenes/Home/Cells/TestRegistration/HomeTestRegistrationCellModel.swift]:
```suggestion
	var gradientViewType: GradientView.GradientType = .lightBlueToWhite 
```

**Pull request merged**